components/general/InfoBlock/InfoBlock.tsx

```diff
@@ -10,6 +10,7 @@ type InfoProps = {
   blockType: InfoBlockType;
   content: string;
   buttonContent?: string;
+  isPhoneSize: boolean;
```

Comment:
![image](https://user-images.githubusercontent.com/73293255/225638921-fe3c6b96-38bd-4cca-be1d-e0bb87245065.png)
Make it optional, so you need not change all those components which based on this one.